db/institutions.yml

```diff
@@ -1401,3 +1401,42 @@ zhejiangu:
   updated: 2020-02-14 19:37:28.925621
   uuid: 4f88b3c9-4719-4a0e-836b-16af6ab6605f
   year: 2020
+icl:
+  aka:
+    - Imperial College
+  name: Imperial College London
+  city: London
+  country: England
+umanchester:
+  aka:
+    - University of Manchester
+  name: The University of Manchester
+  city: Machester
+  country: England
+atladevices:
+  name: Alta Devices
+  city: Sunnyvale
+  country: USA
+  zip: 94085
+  state: California
+mckinseyswitzerland:
+  aka:
+    - McKinsey & Company, Geneva
+  name: McKinsey & Company, Inc. Switzerland
+  city: Genova
+  country: Switzerland
+desy:
+  name: Deutsches Elektronen Synchrotron
+  departments:
+    petra:
+      name: PETRA III
+    cfel:
+      name: Center for Free-Electron Laser Science
+    flash:
+      name: Free electron laser FLASH
+  city: Hamburg
+  country: Deutschland
```

Comment:
haha, nice, but I think that to be consistent we should put here Germany

Reply:
ups.. I guess I was too tired yesterday hahah